src/csharp/Microsoft.Spark.UnitTest/JsonSerDeTests.cs

```diff
@@ -0,0 +1,185 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Text.Json;
+using Microsoft.Spark.Interop.Ipc;
+using Xunit;
+
+namespace Microsoft.Spark.UnitTest
+{
+    public class JsonSerDeTests
+    {
+        [Fact]
+        public void TestSortProperties()
+        {
+            // Arrange
+            var testJson = GetTestJson();
+            using (var document = JsonDocument.Parse(testJson))
+            {
+                var testElement = document.RootElement;
+                var expectedJson = GetExpectedSortedJson();
```

Comment:
How about creating a test json and expected json in a tuple so that it's easier to compare them?